purchase_mass_mail/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
I usually remove copyright lines from `__init__.py` files that just contain imports. But if you leave it, then there's this bug:

```suggestion
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
```